README.md

```diff
@@ -1,3 +1,46 @@
 # API-Monitor
 [![CircleCI](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop.svg?style=svg)](https://circleci.com/gh/Ashaba/API-Monitor/tree/develop)
 [![Coverage Status](https://coveralls.io/repos/github/Ashaba/API-Monitor/badge.svg?branch=develop)](https://coveralls.io/github/Ashaba/API-Monitor?branch=develop)
+ 
+ An application the performs health checks of an api
+ 
+ # Dependencies
+ - `Python 3.6.2`
+ - `PostgreSQL`
+ - `JQuery-3.3.1`
+ - `Flask 0.12.2`
+ 
+ # Installation
+ - Clone the repository  
+ ```$ git clone https://github.com/Ashaba/API-Monitor.git```
+ 
+ - Copy contents of  `env.sample` into `.env`   
+ ```$ cp env.sample .env```
+ 
+ - Configure and setup database in the `.env` file `SQLALCHEMY_DATABASE_URI`  
```

Comment:
I believe the correct grammar is, `Configure the database by modifying the SQLALCHEMY_DATABASE_URI variable in the .env file`